src/SFA.DAS.EmployerAccounts/Commands/OrganisationAndPayeRefData/CookieBasedOrganisationAndPayeDataSaver.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.EmployerAccounts.Interfaces;
+using SFA.DAS.EmployerAccounts.Models.Account;
+
+namespace SFA.DAS.EmployerAccounts.Commands.OrganisationAndPayeRefData
+{
+    public sealed  class CookieBasedOrganisationAndPayeDataSaver : AsyncRequestHandler<SaveOrganisationAndPayeData>
```

Comment:
why not handler?

Reply:
I was following the existing naming convention of the command I'd copied.  I did think about changing it but it would have then been inconsistent.